.github/workflows/actions.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       matrix:
         target:
           - nv14
+          - lr3pro
```

Comment:
It is really not required to add more test targets. This uses already today too many parallel runners on GH.